index.html

```diff
@@ -15,8 +15,14 @@
   </head>
    <body>
    <header>
-        <a class="nav-logo" href="index.html">Emmanuel Kipngeno</a>
-        <nav class="navbar"><svg
+        <nav class="navbar">
```

Comment:
- [ ] Great job on adding the background image as required by the previous reviewer, however while adding it as you may notice, the spacing between your `intro` and the navigation bar is way too big compared to the original design. Kindly make sure you stick to the Figma template. the spacing is supposed to be identical.
- [ ] Also you may consider breaking `I'm Glad to see you` to a new line as is the case on the design.

## Your design :point_down: 
![Screenshot from 2023-05-12 13-30-23](https://github.com/kkmanuu/My-Portfolio/assets/97100091/a5bb0698-7e43-428a-b594-e7eaec510613)
# Figma template:point_down: 
![image](https://github.com/kkmanuu/My-Portfolio/assets/97100091/617a9c71-3892-4fe7-b726-33e47b3f6f27)

Reply:
Done.

Reply:
@christianbiring1 christianbiring1 .Done Making changes